extensions/exposition/components/identity.federation/source/schemas.ts

```diff
@@ -42,4 +42,20 @@ export interface TrustConfiguration {
    * @minItems 1
    */
   audience?: [string, ...string[]];
+  /**
+   * Symmetric encryption secrets
+   */
+  secrets?: {
+    /**
+     * This interface was referenced by `undefined`'s JSON-Schema definition
+     * via the `patternProperty` "^HS\d{3}$".
+     */
+    [k: string]: {
+      /**
+       * This interface was referenced by `undefined`'s JSON-Schema definition
+       * via the `patternProperty` "^\w+$".
+       */
+      [k: string]: string;
+    };
+  };
```

Comment:
I think we should add a `readme.md` to this component, saying that each time the corresponding schema is changed, `pretranspile` script must be executed.

Reply:
Where is it executed?

Reply:
This is NPM built-in feature `pre` and `post` hooks - https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts